src/sctools/test/test_barcode.py

```diff
@@ -1,11 +1,15 @@
 import os
-from .. import barcode, encodings
 import numpy as np
 import pytest
+import pysam
+from .. import barcode, encodings, platform
+
 
 data_dir = os.path.split(__file__)[0] + '/data/'
 
 
+# TEST BARCODES
```

Comment:
Can you divide these tests into different test classes (e.g. `TestBarcodes`, `TestHashErrorsToCorrectBarcodes`, etc.)?

Reply:
I think the answer is "yes, but it'll be complicated": http://computableverse.com/blog/pytest-sharing-class-fixtures. I'm hesitant to do the refactor in this ticket.

Reply:
Ok, I'm not familiar with pytest but this would be a good code-cleanup task since grouping tests into test classes helps keep the code organized, especially as you add in more tests!

Reply:
Definitely agree, but I ran into some pretty significant weirdness when trying it with fixtures. There weren't good examples of how to use fixtures inside classes (since fixtures get passed as parameters, but become methods of cls...) Once that gets sorted out, we can probably fix it. However, it's a 2-point ticket involving a bit of research on how other projects use pytest (and if they use classes, or maybe have their own files, per class that is being tested). 

I've made a ticket.

Reply:
Ok! Thanks for making a ticket!